_commands/basics/history.md

```diff
@@ -3,22 +3,132 @@
 
 history
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`history` is used to display the most recently used commands.
+
 ~~~ bash
-*short* example of command usage and output
+$ history [options]
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+If a number is specified after `history`, up to this number of previous command will 
+be printed. Otherwise, it will default to 500.
+
+#### `history`
+
+~~~ bash
+$ history
+  1  [500th most recently used command]
+  .
+  .
+  500  history
+~~~
 
 ##### Break it down
```

Comment:
Could you use 4 `#`s instead of 5 for all of the "Break it Down"s? A header at the current size looks just as large as the body text, which is a little confusing. Otherwise this is looking pretty solid!

Reply:
er actually never mind, having the Break it Down's being the same size as the command itself could be even more confusing. Instead, could you just make it bold? 👍

Reply:
Done!